tuxemon/prepare.py

```diff
@@ -105,6 +105,11 @@
 # Set the character limit for setting a player name.
 PLAYER_NAME_LIMIT = 30
 
+# Fonts
+FONT_BASIC = "PressStart2P.ttf"
```

Comment:
```if prepare.CONFIG.locale == "zh_CN":
        FONT_FILENAME = prepare.fetch("font", "NotoSansTC.ttf")
    elif prepare.CONFIG.locale == "ja":
        FONT_FILENAME = prepare.fetch("font", "NotoSansJP.ttf")
    else:
        FONT_FILENAME = prepare.fetch("font", "PressStart2P.ttf")
```

Reply:
Yeah, you're right, I didn't understand

Reply:
Still here? Or I need to move somewhere else?

Reply:
I'm sorry, I'm confused, I have no idea what to do.

Reply:
Let's just leave it how it was in your first commit.

Reply:
Actually... We may as well use these consts in theme.py as well.

Reply:
Like this?
```
    if prepare.CONFIG.locale == "zh_CN":
        font_filename = prepare.fetch("font", prepare.FONT_CHINESE)
    elif prepare.CONFIG.locale == "ja":
        font_filename = prepare.fetch("font", prepare.FONT_JAPANESE)
    else:
        font_filename = prepare.fetch("font", prepare.FONT_BASIC)
```

Reply:
Ok, for me no problem, just tell me how do you want theme.py